bugbug/bug_features.py

```diff
@@ -207,6 +207,27 @@ def __call__(self, bug, **kwargs):
         return None
 
 
+class delta_nightly_request_merge(single_bug_feature):
+    def __call__(self, bug, **kwargs):
+        for history in bug["history"]:
+            for change in history["changes"]:
+                if change["added"].startswith("approval-mozilla"):
+                    uplift_request_datetime = datetime.strptime(
+                        history["when"], "%Y-%m-%dT%H:%M:%SZ"
+                    ).replace(tzinfo=timezone.utc)
```

Comment:
Instead of manually parsing with `strptime`, you can use `dateutil.parser.parse`